### ISSUE TITLE
[4th viz] FE create / adapt viz component

### ISSUE THREAD

**Issue by seinig** [boxwise/boxtribute#911]

**Comment by aerinsol** [comment]
Agreed on 2023/9/4 that this should be a Sankey diagram of where items flow to in HERMINE. 

@MaikNeubert, I already took some time to take a look at this some weeks ago, and there is no direct VisX support for Sankey diagrams. However there is quite a lot of interest in the topic and you might be able to get some help / PR review support in the department from the open source community. 

Some nice starting points I found: 
https://github.com/airbnb/visx/issues/190
https://codesandbox.io/s/visx-sankey-chart-poc-forked-mz4f7
https://visx-rodeo.netlify.app/#sankey (click "repo" to see code)


We will still need to talk about the actual requirements, but I hope this is a helpful starting point.

**Comment by aerinsol** [comment]
![Image](https://github.com/boxwise/boxtribute/assets/6267028/71769383-9032-442b-b8eb-a39af3c55d7f)

I used [Sankeymatic](https://sankeymatic.com/build/) (see source code [here](https://github.com/nowthis/sankeymatic) to build this mockup for IHA. It seems very useable and intuitive, and is also based off of D3.  Maybe it will be useful.
